Makefile

```diff
@@ -80,6 +80,10 @@ test: start_trino_if_missing link_to_driver update_deps_files
 	@echo "Testing Starburst driver..."
 	cd $(makefile_dir)/metabase/; DRIVERS=starburst MB_STARBURST_TEST_PORT=$(trino_port) clojure -X:dev:drivers:drivers-dev:test
 
+testOptimized: start_trino_if_missing link_to_driver update_deps_files
```

Comment:
nit: Can we flip them? Can we add `testLegacy` and then move `-J-DexplicitPrepare=false` under `test`? We want this to be the default right? 

Also, whatever we decide, we need to make sure the expected tests are launched here: https://github.com/starburstdata/metabase-driver/blob/main/.github/workflows/ci.yml#L57

Reply:
There was a change, and the new flag is NOT used by default. So `make test` should not be changed and run the legacy flag as usual (one of the Starburst tests does tests both flags), with the option to run the new flag if need be.